src/main.rs

```diff
@@ -8,9 +10,9 @@ mod slack_message;
 
 #[tokio::main]
 async fn main() {
-    tracing_subscriber::fmt()
-        .with_max_level(tracing::Level::INFO)
-        .with_ansi(true)
+    tracing_subscriber::registry()
+        .with(tracing_subscriber::fmt::layer().with_ansi(std::env::var("TERM").is_ok()))
+        .with(tracing_subscriber::EnvFilter::from_env("PIZZAPICKER_LOG"))
```

Comment:
why not just
```rs
tracing_subscriber::registry()
    .with(fmt::layer())
    .with(EnvFilter::from_default_env())
    .init();
```

which uses the standard `RUST_LOG` envvar instead of our own

Reply:
Sure, can do that

Reply:
nice!! ::))